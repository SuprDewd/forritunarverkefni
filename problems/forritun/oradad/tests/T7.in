71 -62 11 99
