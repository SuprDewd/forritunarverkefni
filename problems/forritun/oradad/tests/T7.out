11 -62 71 99
